flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -115,9 +115,20 @@ static TableEnvironment create(EnvironmentSettings settings) {
 	 *
 	 * @param name The name under which the table will be registered.
 	 * @param table The table to register.
+	 * @deprecated use {@link #createTemporaryView(String, Table)}
 	 */
+	@Deprecated
 	void registerTable(String name, Table table);
 
+	/**
+	 * Registers a {@link Table}, which is a view in a SQL terms as a Temporary View.
```

Comment:
The description of this method is tricky, maybe:

`Registers a {@link Table} API object as a temporary view similar to SQL temporary views.`